modules/mono/editor/bindings_generator.cpp

```diff
@@ -3083,6 +3090,9 @@ bool BindingsGenerator::_arg_default_value_from_variant(const Variant &p_val, Ar
 			r_iarg.default_argument = "null";
 			break;
 		case Variant::ARRAY:
+			r_iarg.default_argument = "new %s[] {}";
+			r_iarg.def_param_mode = ArgumentInterface::NULLABLE_REF;
+			break;
```

Comment:
This results in dozens of bogus code being generated:

```cs
[GodotMethod("execute")]
public static int Execute(string path, string[] arguments, Godot.Collections.Array output = null, bool readStderr = false)
{
    Godot.Collections.Array output_in = output != null ? output : new Godot.Collections.Array[] {};
    return NativeCalls.godot_icall_4_1459(__method_bind_10, ptr, path, arguments, output_in.GetPtr(), readStderr);
}
```

`Generated/GodotObjects/OS.cs(350,49): error CS0173: Type of conditional expression cannot be determined because there is no implicit conversion between 'Godot.Collections.Array' and 'Godot.Collections.Array[]'`

Reply:
Fixed.